### PR TITLE
fix: update ember try and travis build configuration

### PR DESCRIPTION
**Pull request by zachnthebox**

### CODE REVIEW

**Comment by zachnthebox** [comment]
Ember try does not use the project's lockfile, and one of the latest dependencies caused the build to fail even though it passes with the current lockfile. 

https://github.com/ember-cli/ember-try/issues/196 starts the discussion on having the option to use the lockfile on Ember Try.

File: tests/helpers/keyboard-helpers.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const keyEvent = function keyEvent(attributes, type, element) {
       event.button = getMouseCode(attribute);
     } else {
       event.key = getKeyCode(attribute);
+      event.keyCode = getKeyCode(attribute);
```
Comment (fdlane) [inline tests/helpers/keyboard-helpers.js]: Can you explain why this is needed please.

File: .travis.yml [hunk 2]
```diff
@@ -1,19 +1,16 @@
 ---
 language: node_js
 node_js:
-  # we recommend testing addons with the same minimum supported node version as Ember CLI
-  # so that your addon works for all apps
-  - "4"
+  - "6"
 
-sudo: false
+sudo: true
```
Comment (fdlane) [inline .travis.yml]: Help me understand this one.  It seems like a critical change to make.

Comment (zachnthebox) [reply .travis.yml]: The Chrome headless browser started failing in the travis environment. The `sudo` option is for the travis build to correctly use Chrome.

You can see a relevant Github issue here - https://github.com/travis-ci/travis-ci/issues/8836#issuecomment-348227535

**Pull request merged**